src/pages/product/Product.jsx

```diff
@@ -0,0 +1,151 @@
+import React, { useEffect, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { useDispatch } from 'react-redux'
+import { saveProductAction } from '../../redux/actions/product'
+import { Card, Button, Select, Input, Table, Popconfirm } from 'antd';
+import { PlusSquareOutlined, SearchOutlined } from '@ant-design/icons';
+import { reqProductList } from '../../api'
+import { reqSearchProduct } from '../../api'
+import { pageSize } from '../../config'
+import { reProductData } from '../../components/reProductData'
+
+
+export default function Product() {
+  const { Option } = Select;
+  const navigate = useNavigate()
+  const [productList, setProductList] = useState([])
+  const [keywords, setKeywords] = useState('')
+  const [isSearch, setIsSearch] = useState(false)
```

Comment:
You can simply use `keywords` state to check if searching products or not. No need to add another state `isSearch` here.